modules/ROOT/pages/amazon/amazon-s3-connector-examples.adoc

```diff
@@ -0,0 +1,155 @@
+= Amazon S3 Examples - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+The following are common use cases for the Amazon S3 connector:
+
+* Store an image from a URL on Amazon S3, and then retrieve and display the image.
+* Create an image link in Amazon S3 and update the status in Twitter along with the image link (not shown in this document).
+
+[[store-and-retrieve]]
+== Use Case: Store and Retrieve an Image from a URL to Amazon S3
+
+Store an image from a URL in Amazon S3, and retrieve and display the image.
+For this example to work in Anypoint Studio, you must provide Amazon Web Services credentials. You can either replace the variables with their values in the code, or you can provide the values for each variable in the `src/main/resources/mule-artifact.properties` file.
+
+image::amazon/amazon-s3-02.png[Studio 7 Visual Studio Icon Flow]
+
+. Create a new Mule project in Studio.
+. Drag an HTTP connector into the canvas, and select it to open the properties editor.
```

Comment:
This is a Mule 3 statement -- in Studio 7, you drag a connector operation to the canvas.